kuma/api/v1/search/__init__.py

```diff
@@ -153,10 +153,13 @@ def _find(params, total_only=False, make_suggestions=False, min_suggestion_score
             # This can happen if the search happened right as the index had
             # just been deleted due to a fresh re-indexing happening in Yari.
             exceptions.NotFoundError,
+            # This can happen when the index simply isn't ready yet.
+            exceptions.TransportError,
         ),
         # The default in redo is 60 seconds. Let's tone that down.
-        "sleeptime": 1,
-        "attempts": 5,
+        "sleeptime": settings.ES_RETRY_SLEEPTIME,
+        "attempts": settings.ES_RETRY_ATTEMPTS,
+        "jitter": settings.ES_RETRY_JITTER,
```

Comment:
I should have done this the first time around. 

Yes, this is unrelated to the (original) issue at hand.